### PR TITLE
Researcher Backfill Notification API

### PR DESCRIPTION
**Pull request by DwayneJengSage**
This is a researcher API to backfill SMS notification registrations for a user. We generally prefer the app register notifications, but sometimes the work can't be done on time, so we want researchers to have the option of backfilling these.

Testing done: added unit tests and integ tests

### CODE REVIEW

**Comment by DwayneJengSage** [comment]
I just realized we should also have a check to make sure the user is consented.

**Review by alxdarksage — APPROVED** [review]

File: conf/routes [hunk 1]
```diff
@@ -96,6 +96,7 @@ GET    /v3/participants/:userId/requestInfo                            @org.sage
...
 GET    /v3/participants/:userId/activities/:activityType/:referentGuid @org.sagebionetworks.bridge.play.controllers.ParticipantController.getActivityHistoryV3(userId: String, activityType: String, referentGuid: String, scheduledOnStart: String ?= null, scheduledOnEnd: String ?= null, offsetKey: String ?= null, pageSize: String ?= null)
 GET    /v3/participants/:userId/activities/:activityGuid               @org.sagebionetworks.bridge.play.controllers.ParticipantController.getActivityHistoryV2(userId: String, activityGuid: String, scheduledOnStart: String ?= null, scheduledOnEnd: String ?= null, offsetBy: String ?= null, offsetKey: String ?= null, pageSize: String ?= null)
+POST   /v3/participants/:userId/createSmsNotificationRegistration      @org.sagebionetworks.bridge.play.controllers.ParticipantController.createSmsNotificationRegistration(userId: String)
 GET    /v3/participants/:userId/notifications                          @org.sagebionetworks.bridge.play.controllers.ParticipantController.getNotificationRegistrations(userId: String)
```
Comment (alxdarksage) [inline conf/routes]: POST /v3/participants/:userId/notifications would be more consistent

**Comment by alxdarksage** [comment]
Approved, but one comment on the URL being used.

File: conf/routes (hunk 1 above)
Comment (DwayneJengSage) [reply conf/routes]: As per IM converstion, we are renaming this to POST /v3/participants/:userId/notifications/sms and we're moving this to an internal-only API.

**Comment by alxdarksage** [comment]
Approved to merge

**Pull request merged**